app/src/main/java/com/example/myattendance/ui/home/HomeScreen.kt

```diff
@@ -52,18 +55,26 @@ fun HomeScreen(
     modifier: Modifier = Modifier,
     @StringRes title: Int
 ) {
+    // to hold state of BottomSheet
+    var isSheetOpen = rememberSaveable {
+        mutableStateOf(false)
+    }
     Scaffold(
+        floatingActionButton = { AppFAB { isSheetOpen.value = true } },
     ) {
         Column(
             modifier = Modifier.padding(it),
             verticalArrangement = Arrangement.spacedBy(4.dp)
         ) {
-            // DailyMedications(navController, analyticsHelper, state, viewModel, navigateToMedicationDetail)
-            DatesHeader({})
+            DatesHeader { }
+            //BottomSheet to add/edit elements , Triggered on floatingActionButton
+            IteamEntryScreen(isSheetOpen = isSheetOpen)
```

Comment:
So, yeah. It is better to have here` if (isSheetOpen) { AttendanceBottomSheet() }` cause we have to manage state at parent and shouldn't give state to children. Really useful google arcticle from google about that: https://developer.android.com/jetpack/compose/state-hoisting